build/configs/rtl8730e/loadable_apps/defconfig

```diff
@@ -1122,11 +1137,14 @@ CONFIG_PM_DOMAIN_NAME_SIZE=32
 CONFIG_DEBUG=y
 CONFIG_DEBUG_ERROR=y
 # CONFIG_DEBUG_WARN is not set
-# CONFIG_DEBUG_VERBOSE is not set
+CONFIG_DEBUG_VERBOSE=y
```

Comment:
lets not enable it by default. user can do in his build environment

Reply:
Done